### PR TITLE
The Slider Widget

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md [hunk 1]
```diff
@@ -87,8 +84,19 @@ You can click the image to zoom in, where you can browse through the images in t
 
 ![movePage.jpg](images/The-Slider-Widget.png)
 
-The Slider Widget allows you to pick multiple images and then make them into a slideshow, you will have the ability to choose how fast the transition between images is.
+The Slider Widget allows you to pick multiple images and then make them into a slideshow, you will have the ability to choose how fast the transition between images is here. It is important to know that the value 1000 is 1 second.
```
Comment (sofietoft) [inline Add-ons/Starter-Kit-TEMP/Widgets/index.md]:
```suggestion
The Slider Widget allows you to pick 1 or more widgets to display in a slideshow. 

You will have the ability to choose how fast the transition between widgets is here. It is important to know that 1000 is equal to 1 second.
```

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md [hunk 2]
```diff
@@ -87,8 +84,19 @@ You can click the image to zoom in, where you can browse through the images in t
...
+
+You will have the option to customize the slider to your liking, by deciding if you want the slider to have arrows on the edges which takes you through the slides. If you want dots in the button to show what slide you are currently on. You can make the slider pause when you hover over it. You will have to pick what the slider displays by picking the Widgets. You can pick multiple Widgets.
```
Comment (sofietoft) [inline Add-ons/Starter-Kit-TEMP/Widgets/index.md]: Could we make this into a list instead? Would be a quicker way to get an overview of all the settings that can be customized.

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md [hunk 3]
```diff
@@ -87,8 +84,19 @@ You can click the image to zoom in, where you can browse through the images in t
 
 ![movePage.jpg](images/The-Slider-Widget.png)
 
-The Slider Widget allows you to pick multiple images and then make them into a slideshow, you will have the ability to choose how fast the transition between images is.
+The Slider Widget allows you to pick multiple images and then make them into a slideshow, you will have the ability to choose how fast the transition between images is here. It is important to know that the value 1000 is 1 second.
+
+You will have the option to customize the slider to your liking, by deciding if you want the slider to have arrows on the edges which takes you through the slides. If you want dots in the button to show what slide you are currently on. You can make the slider pause when you hover over it. You will have to pick what the slider displays by picking the Widgets. You can pick multiple Widgets. 
+
+You can choose to have the slider to ***Wrap Around*** to make it loop around indefinitely, and to make the transition between slides more seamless you can add the ***Fade Animation***
```
Comment (sofietoft) [inline Add-ons/Starter-Kit-TEMP/Widgets/index.md]:
```suggestion
You can choose to have the slider ***Wrap Around*** to make it loop between the widgets indefinitely.  You can also make the transition between slides more seamless by using ***Fade Animation***.
```

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md [hunk 4]
```diff
@@ -87,8 +84,19 @@ You can click the image to zoom in, where you can browse through the images in t
...
+
+See below an example of how a slider could look, it contains a ***Image Widget***,it uses ***Show Next/prev arrows*** and ***Show Page Dots***.
```
Comment (sofietoft) [inline Add-ons/Starter-Kit-TEMP/Widgets/index.md]:
```suggestion
Below is an example of how a slider could look. It contains an ***Image Widget***, uses ***Show Next/prev arrows*** and ***Show Page Dots***.
```

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md [hunk 5]
```diff
@@ -87,8 +84,19 @@ You can click the image to zoom in, where you can browse through the images in t
...
+It has a black ***Background Color*** picked in the ***Image Widget***.   
+
+![movePage.jpg](images/Slider-example.png)
 
+::: Note 
+For the best experience with an image slider add multiple ***Image Widgets*** containing one image each.:::
```
Comment (sofietoft) [inline Add-ons/Starter-Kit-TEMP/Widgets/index.md]:
```suggestion
For the best experience with the Slider Widget, we recommend adding multiple ***Image Widgets*** containing one image each.
:::
```

File: Add-ons/Starter-Kit-TEMP/Widgets/index.md (hunk 2 above)
Comment (ctf-umbraco) [reply Add-ons/Starter-Kit-TEMP/Widgets/index.md]: ye that would make good sense

**Pull request merged**